pandas/tests/indexing/multiindex/test_partial.py

```diff
@@ -208,6 +215,30 @@ def test_setitem_multiple_partial(self, multiindex_dataframe_random_data):
         expected.loc["bar"] = 0
         tm.assert_series_equal(result, expected)
 
+    def test_partial_getitem_loc_datetime(self):
+        # GH: 25165
+        date_idx = date_range("2019", periods=2, freq="MS")
+        df = DataFrame(
+            list(range(4)),
+            index=MultiIndex.from_product([date_idx, [0, 1]], names=["x", "y"]),
+        )
+        expected = DataFrame(
+            [2, 3],
+            index=MultiIndex.from_product(
+                [[to_datetime("2019-02-01")], [0, 1]], names=["x", "y"]
+            ),
+        )
+        result = df["2019-2":]
```

Comment:
Some more cases to check:

does all of this work if you index on `df2 = df.swaplevel(0, 1)` and index with `df2.loc[:, "2019-02":]`?  how about `df2.loc[:, "2019-02":, :]`?

how about flipping the side of the slide from `"2019-02":` to `:"2019-02"`?

Side-note: looks like without the slice `df.loc["2019-02"]` works but `df.loc[:, "2019-02"]` raises `KeyError`

Reply:
``df2.loc[:, "2019-02":]`` does not work, because ``2019-02:`` is interpreted as columns. Is this appropriate?
``df2.loc[:, "2019-02":, :]`` works after fixing another bug and sorting the index of course
Flipping works as expected

``df.loc[:, "2019-02"]`` interprets ``2019-02`` as column

Reply:
thanks